bin/node-template/pallets/template/src/mock.rs

```diff
@@ -36,7 +36,7 @@ impl system::Config for Test {
 	type AccountId = u64;
 	type Lookup = IdentityLookup<Self::AccountId>;
 	type Header = Header;
-	type Event = Event;
+	type Event = RuntimeEvent;
```

Comment:
What should happen here is that the associated type should also be renamed to `RuntimeEvent`, so that it's clear which `Event` type to use while configuring the pallet.